manifests/bases/hr/deployment.yaml

```diff
@@ -23,8 +23,8 @@ spec:
         - name: github-container-registry
       containers:
         - name: hr
-          image: ghcr.io/redbadger/hr
-          imagePullPolicy: Never
+          image: hr
```

Comment:
What's the reason for doing this?

Reply:
Hi David, there are a few things here

1. The kustomization.yaml files match on the "image" value I think (despite it looking like they match on "name"). So matching on 'hr' is a bit more intuitive than matching on ghcr.io/redbadger/hr
2. Using 'hr' makes it a bit more obvious that the name needs to be replaced in a kustomization.yaml
3. The imagePullPolicy now defaults to something that would work in production but fail locally, which is probably the right way around (the opposite can lead to disasters!)